nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -297,7 +300,7 @@ private Tree suppressibleNode(@Nullable TreePath path) {
       Tree suggestTree, Description.Builder builder) {
     assert suggestTree.getKind() == Tree.Kind.METHOD_INVOCATION;
     final MethodInvocationTree invTree = (MethodInvocationTree) suggestTree;
-    final Symbol.MethodSymbol methodSymbol = ASTHelpers.getSymbol(invTree);
+    final Symbol.MethodSymbol methodSymbol = castToNonNull(ASTHelpers.getSymbol(invTree));
```

Comment:
If we update to Error Prone 2.11.0 first, we can get rid of many `castToNonNull` calls as certain `ASTHelpers.getSymbol()` overloads no longer return `@Nullable`; see https://github.com/google/error-prone/commit/c2e14f24facfae7a0e38e78dd4005f25c7f4f8b4